src/templates/article.js

```diff
@@ -100,13 +103,42 @@ const Icons = styled('div')`
         }
     }
 `;
+
 const Container = styled('div')`
-    margin: 0 auto;
+    display: grid;
```

Comment:
For mobile, can we give a 16px margin on the left and right sides? The content should have a bit of room before the edge (as it seems to indicate on the Figma)